Sources/Rendering/Core/Viewport/index.d.ts

```diff
@@ -65,7 +65,7 @@ export interface vtkViewport extends vtkObject {
   /**
    *
    */
-  getViewport(): vtkViewport;
+  getViewport(): [number, number, number, number];
```

Comment:
For consistency, you must do the same with `getViewportByReference`.

Can you take the opportunity to improve the documentation:

```
/**
 * Viewport for the Viewport to draw in the rendering window.
 * Coordinates are expressed as (xmin,ymin,xmax,ymax), where each
 * coordinate is 0 <= coordinate <= 1.0.
 * @returns {Number[]}
 * @see getViewportByReference()
 * @see setViewport()
 */
 ```